flink-clients/src/main/java/org/apache/flink/client/program/Client.java

```diff
@@ -491,9 +495,58 @@ public void cancel(JobID jobId) throws Exception {
 		}
 	}
 
+	/**
+	 * Stops a program on Flink cluster whose job-manager is configured in this client's configuration.
+	 * Stopping works only for streaming programs. Be aware, that the program might continue to run for
+	 * a while after sending the stop command, because after sources stopped to emit data all operators
+	 * need to finish processing.
+	 * 
+	 * @param jobId
+	 *            the job ID of the streaming program to stop
+	 * @throws ProgramStopException
+	 *             If the job ID in invalid (ie, is unknown or refers to a batch job) of if sending the stop signal
+	 *             failed. That might be due to an I/O problem, ie, the job-manager is unreachable.
+	 */
+	public void stop(JobID jobId) throws ProgramStopException {
+		LOG.info("JobManager actor system address is " + jobManagerAddress);
+		
+		LOG.info("Starting client actor system");
+		final ActorSystem actorSystem;
+		try {
+			actorSystem = JobClient.startJobClientActorSystem(configuration);
+		}
+		catch (Exception e) {
```

Comment:
try - catch block formatting is not consistent. Here you insert a line break after the `}` and in another block you only insert a whitespace. I'm slightly in favour of the `} catch (...) {` format.